rtloader/common/builtins/util.c

```diff
@@ -8,14 +8,18 @@
 
 #include <stringutils.h>
 
+static cb_obfuscate_sql_t cb_obfuscate_sql = NULL;
+
 static PyObject *headers(PyObject *self, PyObject *args, PyObject *kwargs);
 static PyObject *get_hostname(PyObject *self, PyObject *args);
 static PyObject *get_clustername(PyObject *self, PyObject *args);
 static PyObject *log_message(PyObject *self, PyObject *args);
 static PyObject *set_external_tags(PyObject *self, PyObject *args);
+static PyObject *obfuscate_sql(PyObject *self, PyObject *args);
 
 static PyMethodDef methods[] = {
     { "headers", (PyCFunction)headers, METH_VARARGS | METH_KEYWORDS, "Get standard set of HTTP headers." },
+    { "obfuscate_sql", (PyCFunction)obfuscate_sql, METH_VARARGS, "Obfuscate Sql." },
```

Comment:
This should be move to the `datadog_agent` builtin. `util` package is a legacy requirement from A5.